chapter08/jxp/ch8ex1.pl

```diff
@@ -0,0 +1,30 @@
+use strict;
+use warnings;
+use IO::Tee;
+
+my($sec, $min, $hour, $day, $mon, $yr, $dow) = localtime;
```

Comment:
You might want to take a look at `Time::Piece` (http://perldoc.perl.org/Time/Piece.html) - it overrides `localtime` to return an object on which you can call simple methods to obtain the date components.